cmd.c

```diff
@@ -758,28 +757,41 @@ void cmd_app(char *s)
 	phfs.dataOffs = 0;
 
 	/* Parse command arguments */
-	if (cmd_parseArgs(s, cmdArgs, &cmdArgsc) < 0) {
-		plostd_printf(ATTR_ERROR, "\nWrong arguments!!\n");
+	if (cmd_parseArgs(s, cmdArgs, &cmdArgsc) < 0 && (cmdArgsc < 3 || cmdArgsc > 6))
+		goto errWrongArgs;
```

Comment:
Look [PR #6](https://github.com/phoenix-rtos/phoenix-rtos-doc/pull/6)

Reply:
,, Wrong arguments '' ?

Reply:
I meant the use of `goto`.

Reply:
I meant my arguments presented here ,, [PR #6](https://github.com/phoenix-rtos/phoenix-rtos-doc/pull/6) ''.